<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poker_Server</class>
 <widget class="QWidget" name="Poker_Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PokerServer</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_log">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_lelaoji">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_listen">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>本地测试</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_money">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>每人默认底注：</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>哪位玩家判断输赢：</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_listen_net">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>网络监听</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示所有人的积分</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
